sched/queue/queue.go

```diff
@@ -0,0 +1,40 @@
+package queue
+
+import (
+	"github.com/scootdev/scoot/sched"
+	"time"
+)
+
+// Note: queued jobs can be read straight out of a channel that should
+// be returned along with the creator of a Queue
+type Queue interface {
```

Comment:
Yeah, that makes sense.

Now the channel of work is of type queue.WorkItem, which includes a Job and a Dequeue method.

How's that?